src/geointerface.jl

```diff
@@ -142,6 +103,14 @@ let pointtypes = (wkbPoint, wkbPoint25D, wkbPointM, wkbPointZM),
         return getgeom(geom, i - 1)
     end
 
+    function GeoInterface.getpoint(
+        ::GeoInterface.AbstractLineStringTrait,
+        geom::AbstractGeometry{wkbLineString},
+    )
+        refs = Ref{Float64}(), Ref{Float64}(), Ref{Float64}()
+        (getpoint!(geom, i, refs...) for i in (0:GeoInterface.npoint(geom)-1))
+    end
```

Comment:
Should we add a unit test for this?

Reply:
Yep. I'll actually put it in a separate PR. It doesn't really belong here, I just profiled it all at once.